controllers/v1/saved_jobs/index.js

```diff
@@ -32,10 +32,10 @@ exports.createSavedJobs = async (req, res, next) => {
     console.log('no job found')
     }
 
-    const SavedJobss = await SavedJobs.find({}).populate("user_id").populate("job_id").sort("-createdAt");
+    const SavedJobss = await SavedJobs.find({user_id: req.user.id}).populate("user_id").populate("job_id").sort("-createdAt");
 
     SavedJobss.map((saved) => {
-      if (saved.job_id === req.body.job_id && saved.user_id === req.user.id){
+      if (saved.job_id === req.body.job_id){
```

Comment:
put this in toString(), otherwise, it won't work!! for example `(saved.job_id).toString() `
check joinChapter endpoints